crates/miden-lib/asm/miden/contracts/auth/basic.masm

```diff
@@ -74,6 +75,40 @@ export.auth__tx_rpo_falcon512
     # AS => []
 end
 
+#! Inputs:  [SALT, OUTPUT_NOTES_COMMITMENT, INPUT_NOTES_COMMITMENT, ACCOUNT_DELTA_COMMITMENT]
+#! Outputs: [SALT, OUTPUT_NOTES_COMMITMENT, INPUT_NOTES_COMMITMENT, ACCOUNT_DELTA_COMMITMENT]
+proc.adv_insert_hqword.16
+    loc_storew.0
+    movdnw.3
+    loc_storew.4
+    movdnw.3
+    loc_storew.8
+    movdnw.3
+    loc_storew.12
+    movdnw.3
+    # => [SALT, OUTPUT_NOTES_COMMITMENT, INPUT_NOTES_COMMITMENT, ACCOUNT_DELTA_COMMITMENT]
+
+    exec.hash_tx_summary
+    # => [MESSAGE]
```

Comment:
this is doing hashing "again" (it's done later in the main `auth__` procedure), but we need `MESSAGE` to place the commitments under the right key in the map.
The way it's done now, the call to `adv_insert_hqword` would be replaced by the `adv.` instruction with the exact same "signature" which is neat.